### PR TITLE
Chore/jest configurator author profile

### PR DESCRIPTION
**Pull request by nikhedonia**
This Branch uses jest-configurator for testing author profile.

I've merged https://github.com/newsuk/times-components/pull/613 into this branch to resolve the jest-configurator issue.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/39c97d7677e600e57960abe6cf736aa9065ddbcb/dextrose-test/index.html

File: packages/author-profile/package.json [hunk 1]
```diff
@@ -4,15 +4,14 @@
...
     "prettier:diff": "prettier --list-different '**/*.*'",
-    "lint": "eslint . && yarn prettier:diff",
-    "dextrose-stories": "dextrose-stories",
-    "dextrose-clean": "dextrose-clean"
+    "lint": "eslint . && npm run prettier:diff",
+    "test": "yarn test:android --ci --bail --coverage && yarn test:ios --ci --bail --coverage && yarn test:web --ci --bail --coverage",
+    "test:android": "jest --no-cache --config='./__tests__/android/jest.config.js'",
+    "test:ios": "jest --no-cache --config='./__tests__/ios/jest.config.js'",
+    "test:web": "jest --no-cache --config='./__tests__/web/jest.config.js'",
+    "test:watch": "jest --projects */**/jest.config.js --watchAll"
```
Comment (jimhunty) [inline packages/author-profile/package.json]: Remove the watch tests

File: packages/author-profile/__tests__/author-profile-content.native.js [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+    render() {
+      return <MockScrollView {...this.props} />;
+    }
+  }
+  ScrollView.propTypes = RealScrollView.propTypes;
+  return ScrollView;
+});
+
+Enzyme.configure({ adapter: new React16Adapter() });
```
Comment (craigbilner) [inline packages/author-profile/__tests__/author-profile-content.native.js]: shouldn't need this?

File: packages/author-profile/__tests__/ios/author-profile.test.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import native from "../author-profile-content.native";
+
+describe("AuthorProfile test on ios: ", () => {
```
Comment (craigbilner) [inline packages/author-profile/__tests__/ios/author-profile.test.js]: I think there's a very casual syntax for this followed by the other packages....

File: packages/utils/graphql.js [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import React, { Component } from "react";
 import ApolloClient from "apollo-client";
 import { ApolloProvider } from "react-apollo-temp";
-import { MockLink } from "react-apollo-temp/lib/test-links";
+import { MockLink } from "react-apollo-temp/lib/test-utils";
```
Comment (craigbilner) [inline packages/utils/graphql.js]: this has now been merged

File: packages/author-profile/__tests__/author-profile-content.native.js [hunk 5]
```diff
@@ -0,0 +1,256 @@
+import "react-native";
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import React16Adapter from "enzyme-adapter-react-16";
```
Comment (colincclark) [inline packages/author-profile/__tests__/author-profile-content.native.js]: this is all in `jest-configurator` is it not?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4be59ef1249ac5b6617004f9b1363f0e19c2998d/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/188e7fd7ca27854f42056702b1d54ed33c82dff1/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/2d5cd955c0eb4461c98d885ec3347deea195c92d/dextrose-test/index.html

File: packages/author-profile/__tests__/author-profile-content.native.js (hunk 5 above)
Comment (nikhedonia) [reply packages/author-profile/__tests__/author-profile-content.native.js]: don't think so, get a bunch of errors if i leave those out

Comment (colincclark) [reply packages/author-profile/__tests__/author-profile-content.native.js]: @nikhedonia but you removed it, and the tests passed....? approved

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/fd5ca9ba520ed902f7ee4a4ce3740dce96fc262c/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/146a8d525f6ab78e46311c2114c19a778e60da0e/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c75598ed5e932763337658a9ff17bd2f9d6ba1bd/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7aa21fec9cfc6326bf7ed420a3096ccbddba895a/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7714d33c587f24fe85ce752d217bfb74eb97924f/dextrose-test/index.html

**Pull request merged**